### PR TITLE
Added support for query batching

### PR DESCRIPTION
**Pull request by ronald-d-rogers**

### CODE REVIEW

File: src/main/java/graphql/servlet/GraphQLServlet.java [hunk 1]
```diff
@@ -266,14 +286,42 @@ private GraphQL newGraphQL(GraphQLSchema schema) {
...
+        query(query, operationName, variables, schema, context, rootObject, (r) -> {
+            resp.setContentType(APPLICATION_JSON_UTF8);
+            resp.setStatus(r.getStatus());
+            resp.getWriter().write(r.getResponse());
+        });
+    }
+
+    private void doBatchedQuery(Iterator<GraphQLRequest> graphQLRequests, GraphQLSchema schema, GraphQLContext context, Object rootObject, HttpServletRequest req, HttpServletResponse resp) throws Exception {
+        resp.setContentType(APPLICATION_JSON_UTF8);
+        resp.setStatus(STATUS_OK);
```
Comment (ronald-d-rogers) [inline src/main/java/graphql/servlet/GraphQLServlet.java]: A call to `query` below might result in an exception. Is it ok to set this here?

**Comment by ronald-d-rogers** [comment]
@apottere Have you had a chance to look at this yet? We're actually pining to have this in production here at Bloomberg BNA for one of our projects. We refactored one of our sites to use Apollo not realizing until it was too late that Apollo required server support for query batching.

**Comment by apottere** [comment]
@ronald-d-rogers sorry, I've been swamped lately and just got a chance to review it.  It looks good to me, thanks for the great work on this 👍

**Comment by ronald-d-rogers** [comment]
Thanks! Don't suppose we could get a release? We were just about to make the decision to bundle the forked version into the build but now that this has been merged...

**Comment by ronald-d-rogers** [comment]
I take that back, looks like a release has already been cut!

**Comment by dkubicki** [comment]
@ronald-d-rogers  #295 Have you tried some kind of this approach?

**Pull request merged**